test/unit/lua_func_adapter.c

```diff
@@ -0,0 +1,267 @@
+#include "diag.h"
+#include "fiber.h"
+#include "memory.h"
+#include "lua/init.h"
+#include "lua/error.h"
+#include "lua/utils.h"
+#include "lua/msgpack.h"
+#include "box/tuple.h"
+#include "box/lua/func_adapter.h"
+#include "box/lua/tuple.h"
+#include "core/func_adapter.h"
+
+#define UNIT_TAP_COMPATIBLE 1
+#include "unit.h"
+#include "lua_test_utils.h"
+
+#define EPS 0.0001
+
+/**
+ * Check if two floating point numbers are equal.
+ */
+static bool
+number_eq(double a, double b)
+{
+	return fabs(a - b) < EPS;
+}
+
+#undef EPS
+
+static struct lua_State *L;
+static struct func_adapter_ctx ctx;
```

Comment:
Nit: There's no need for `ctx` to be global. Please define it on stack where it's used.

Reply:
Done.